persistence/sql/identity/persister_identity.go

```diff
@@ -83,9 +130,9 @@ func (p *Persister) normalizeIdentifier(ct identity.CredentialsType, match strin
 	return match
 }
 
-func (p *Persister) FindByCredentialsIdentifier(ctx context.Context, ct identity.CredentialsType, match string) (*identity.Identity, *identity.Credentials, error) {
+func (p *IdentityPersister) FindByCredentialsIdentifier(ctx context.Context, ct identity.CredentialsType, match string) (_ *identity.Identity, _ *identity.Credentials, err error) {
 	ctx, span := p.r.Tracer(ctx).Tracer().Start(ctx, "persistence.sql.FindByCredentialsIdentifier")
-	defer span.End()
+	otelx.End(span, &err)
```

Comment:
Did the `defer` go missing here? @alnr

Reply:
Wow, well spotted!
https://github.com/ory/kratos/pull/3145

Reply:
thank you for double checking @jonas-jonas 🙏